docs/use-linea/explore/use-snapshotx.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Use Snapshot X EVM
+sidebar_position: 1
+---
+
+# Snapshot X
+
+**Snapshot X** is an on-chain voting protocol. Technically speaking, it's a set of modular smart-contracts that interact with each other to do coordinate. The difference with the original [Snapshot](https://snapshot.org) is that Snapshot X is **fully on-chain**.
+
+If anything in these docs is unclear or you would like more detail, do not hesitate to reach out on [Discord](https://discord.gg/snapshot) or view the full documentation at [https://docs.snapshotx.xyz](https://docs.snapshotx.xyz).
+
+Let's quickly go over the core elements of the voting process:
+
+* **Space** - You can think of a space as an organization's account on Snapshot which can be viewed by anyone visiting the platform. It serves as a hub for all proposals related to the organization and a source of information for the users.
+* **Proposal** - Proposal is the key element of the voting system. It presents a change suggestion related to a specific organization and enables eligible users to cast their vote. 
+* **Voting Strategies** - Voting strategies are the contracts used to determine the voting power (VP) of users.
+
+
+# Voting
+
+## Who can vote on proposals? <a href="#who-can-vote-on-proposals" id="who-can-vote-on-proposals"></a>
+
+Each space specifies their [Voting Strategies](https://docs.snapshotx.xyz/protocol-sx-evm/voting-strategies.md) in its settings. You can see the custom setup by opening the space settings. This setup can define if you are eligible to take part in the voting and what is your Voting Power calculated at the timestamp of proposal creation.
+
+In case of Linea at the time of the quest every address has 1 Voting Power.
+
+
+# Quest - Cast a vote <a href="#cast-a-vote" id="cast-a-vote"></a>
+
+{% hint style="info" %}
+On-chain voting on Snapshot X doesn't cost you anything nor does it affect your funds in any way.
+{% endhint %}
```

Comment:
this is not a real word

can you update `project-words.txt` to allowlist non-real words?
You can see the failures here:

<img width="914" alt="Screenshot 2023-06-13 at 10 06 08 AM" src="https://github.com/ConsenSys/doc.zk-evm/assets/17280653/2f5dbec1-2ff6-4f2c-b67d-5a7d8d39b3e7">

And then run `npm run lint` afterwards to check?

lmk if you need help

Reply:
Added 👌 
Just to make sure with the `{% hint` syntax - you are using Gitbook right? If so it should render it correctly.